pom.xml

```diff
@@ -163,28 +163,38 @@
     <dependencyManagement>
         <dependencies>
             <dependency>
-                <groupId>javax.enterprise</groupId>
-                <artifactId>cdi-api</artifactId>
+                <groupId>jakarta.enterprise</groupId>
+                <artifactId>jakarta.enterprise.cdi-api</artifactId>
                 <version>${cdi-version}</version>
+                <exclusions>
```

Comment:
Nice catch on these exclusions!

Reply:
Hey, I just copied and pasted from a prior PR YOU provided me to, LOL. Thanks for the reference!